docs/source/recipes/pandas_vs_fiftyone.rst

```diff
@@ -0,0 +1,256 @@
+___________
+
+.. list-table::
+   :widths: 20 40 40
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - 
+     - **pandas**
+     - **FiftyOne**
+   * - Make a copy
+     - ``df.copy()``
+     - ``ds.clone()`` 
+   * - Slice
+     - ``df[start:end]``
+     - ``ds[start:end]`` 
+   * - Random sample
+     - ``df.sample(n=n)``
+     - ``ds.take(n)``
+   * - Shuffle data
+     - ``df.sample(frac=1)``
+     - ``ds.shuffle()``
+   * - Filter by column/field value
+     - ``df[df[*] > threshold]``
+     - ``ds.match(F(*) > threshold)``
+   * - Sort values
+     - ``df.sort_values()``
+     - ``ds.sort_by(*)``
+   * - Delete all
+     - | ``import gc``
+       | ``del df; gc.collect()``
+     - ``ds.delete()``
+
+Aggregations
+____________
+
+.. list-table::
+   :widths: 20 40 40
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - 
+     - **pandas**
+     - **FiftyOne**
+   * - Count
+     - ``df[*].count()``
+     - ``ds.count(*)`` 
+   * - Sum
+     - ``df[*].sum()``
+     - ``ds.sum(*)`` 
+   * - Unique values
+     - ``df[*].unique()``
+     - ``ds.distinct(*)``
+   * - Bounds
+     - | ``min = df[*].min()``
+       | ``max = df[*].max()``
+     - ``min, max = ds.bounds(*)``
+   * - Mean
+     - ``df[*].mean()``
+     - ``ds.mean(*)``
+   * - Standard deviation
+     - ``df[*].std()``
+     - ``ds.std(*)``
+   * - Quantile
+     - ``df[*].quantile(values)``
+     - ``ds.quantiles(*, values))``
+
+.. note::
+
+    The table above assumes you have imported:
+
+    .. code-block:: python
+
+        import numpy as np
+
+Structural changes
+__________________
+
+.. list-table::
+   :widths: 20 40 40
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - 
+     - **pandas**
+     - **FiftyOne**
+   * - New column/field as constant value
+     - ``df["col"] = value``
+     - | ``ds.add_sample_field("field", fo.StringField)``
+       | ``ds.set_field("field", value).save()``
+   * - New column/field from external data
+     - ``df["col"] = data``
+     - ``ds.set_values("field", data)``
+   * - New column/field from existing columns/fields
+     - ``df["col"] = df.apply(fcn, axis=1)``
+     - | ``ds.add_sample_field("field", fo.FloatField)`` or
+       | ``ds.set_field("field", expression).save()``
+   * - Remove a column/field
+     - ``df = df.drop(["col"], axis=1)``
+     - | ``ds.delete_sample_fields(["field"])`` or
+       | ``ds.exclude_fields(["field"]).keep_fields()``
+   * - Keep only specified columns/fields
+     - ``df["col1", "col2"]``
+     - ``ds.select_fields(["field1", "field2"])``
+   * - Concatenate DataFrames or DatasetViews
+     - ``pd.concat([df1, df2])``
+     - ``view1.concat(view2)``
+   * - Add a single row/sample
+     - ``df.append(row, ignore_index=True)``
+     - ``ds.add_sample(sample)``
+   * - Remove rows/samples
+     - ``df.drop(rows)``
+     - | ``ds.delete_samples(sample_ids)`` or
+       | ``ds.exclude(samples).keep()``
+   * - Keep only specified rows/samples
+     - ``df.iloc[rows]``
+     - ``ds.select(sample_ids)``
```

Comment:
Okay to use sample IDs in the FiftyOne version, because selecting by positional index is not an efficient operation in FiftyOne. To me this evokes the most idiomatic equivalence, which is what we want the takeaway to be